windows/src/developer/TIKE/main/RedistFiles.pas

```diff
@@ -124,8 +124,15 @@ function GetWixPath: string;
 end;
 
 function GetRedistProjectTemplatePath: string;
+var
+  root: string;
+const
+  DevProjectTemplatePath = 'windows\src\developer\kmconvert\data\';
 begin
-  Result := GetDebugPath('Debug_RedistTemplatePath', ExtractFilePath(ParamStr(0))+'projects\templates\');
+  if TKeymanPaths.RunningFromSource(root)
+    then Result := root + DevProjectTemplatePath
+    else Result := ExtractFilePath(ParamStr(0))+'projects\templates\';
+  Result := GetDebugPath('Debug_RedistTemplatePath', Result);
```

Comment:
This allows us to create new projects while debugging Keyman Developer from source, without fiddling with registry settings. I just threw it in here as a rider; it matches similar patterns in RedistFiles.pas and KeymanPaths.pas.